practices/run-pair-programming-sessions.md

```diff
@@ -1,103 +1,129 @@
 # Run Pair Programming Sessions
 
-Run Pair Programming Sessions involve two developers collaborating either at one workstation or remotely, with one typing out code (the driver) and the other reviewing.
-This practice enhances code quality by facilitating constant review, improving knowledge sharing, discussion, and problem-solving.
+In pair programming sessions, two developers collaborate either at one workstation or remotely, with one typing out code (the driver) and the other reviewing (the navigator).
+This practice enhances code quality by facilitating consistent review, knowledge sharing, discussion, and problem-solving.
 It also serves as a mentoring tool.
-When developers don't engage in Pair Programming, bugs increase, knowledge silos form, problem-solving becomes slower, and teams miss opportunities for skill development.
-Effective Pair Programming requires clear communication, mutual respect, regular role rotation, and a supportive team culture that values collaboration and continuous improvement.
+When developers *don't* engage in pair programming, bugs increase, knowledge silos form, problem-solving becomes slower, and teams miss opportunities to develop skills.
+Effective pair programming requires clear communication, mutual respect, regular role rotation, and a supportive team culture that values collaboration and continuous improvement.
 
 ## Nuances
 
+This section outlines common pitfalls, challenges, or limitations teams commonly encounter when applying this practice.
+
+<ed> Consider swapping Nuances and How to Improve sections. This way, we introduce the Practice and then lead with actionable ways to implement it, rather than the pitfalls/challenges of implementation. Now, if I’ve misinterpreted the Nuance section, please let me know. In the Capability, the Nuances were described as pitfalls/limitations/challenges. But, during our initial call in Sept., you mentioned the Nuances as when to use/not use a given practice. After reading the drafts, I understand the Nuances to be the former — pitfalls/limitations/challenges. </ed>
```

Comment:
I like the idea of having nuance go next. My mind tends to think about the practicality of any advice before I am convinced that making an improvement in an area is worth my time. Having an understanding of the nuance helps me frame the advice into my personal situation, thus making that decision easier.

If you feel strongly about this recommendation, I'd be open to discussing this further and better understanding your point of view.

To your point about nuance being about when to use / not use a given practice, that's not always a straight forward answer. The goal is to arm the reader with the right context to make that decision for themselves. If there's ever a time where you don't feel we've made that easy for the reader's I'd like you to call that out and we can add/remove/adjust the content.

Reply:
That makes perfect sense. Given that, I suggest we soften the "intro" statement for Nuances a bit (the line that will go just under the header for Nuances, to introduce the section). Perhaps something like: 

"This section outlines common pitfalls, challenges, or limitations teams commonly encounter when applying this practice. The goal is not to discourage you. Rather, the goal is to arm you with the appropriate context so that you can make an informed decision about when and how to implement the practice with your teams."

Reply:
Edited.